targets/TARGET_Maxim/TARGET_MAX32600/device/TOOLCHAIN_ARM_STD/MAX32600.sct

```diff
@@ -19,8 +19,7 @@ LR_IROM1 0x00000000 0x40000  {    ; load region size_region
         .ANY (+RO)
     }
 
-    ; [RAM] Vector table dynamic copy: 79 vectors * 4 bytes = (0x140) - alignment
-    RW_IRAM1 (0x20000000+0x140) (0x8000-0x140-Stack_Size)  {  ; RW data
+    RW_IRAM1 (0x20000000) (0x8000-Stack_Size)  {  ; RW data
```

Comment:
Why is vector size removed from start of RAM? 
In order to remove --legacyalign every section should be 8-byte aligned and for that I updated the linker scripts (most) to have vectors and all other sections to be 8-byte aligned.

Reply:
Explanation is in commit message: https://github.com/ARMmbed/mbed-os/pull/10373/commits/3fea439b1288c1792b3fa4aef8a67a8a0f3707bd

Not sure why Maxim is being unusual here - alternatively we could change their code to work like all the other targets.

But I do actually prefer Maxim's approach. It seems a lot simpler - just put the vectors in the normal RAM with an align directive.

Only reason I can see for the special linker map handling is that maybe it avoids stupid padding? The vectors do have a high alignment requirement (512-byte or more), so putting them at the front avoids a linker being dumb and sticking them in the middle with a huge padding block in front of them.

But are any of the linkers really that dumb?

Reply:
cc @ARMmbed/team-maximintegrated

Reply:
Or maybe that commit message isn't clear enough.

Because the Maxim build puts its vectors in main RAM, the main RAM needs to be 512-byte aligned, if you want to turn `--legacyalign` off.

Taking that unused vector space out makes the main RAM 512-byte aligned.

The vector space is not there in the IAR and GCC maps.